Automattic/wp-calypso
# Move lib/cart actions and types out of lib/upgrades

Another reorg (after) of `lib/cart/upgrades` actions and action types. This time, we move Cart Store actions to `lib/cart` and update all imports.

This patch breaks up dependencies that should be independent from each other and makes it easy to find where exactly the shopping cart is manipulated.

**How to test:**
1. Verify that one can add stuff to the cart and proceed to checkout. Plenty of places (domains, GSuite, plans, ...) add an item to the cart (`addItem`) and then navigate to `/checkout`.
2. Verify that the cart can be purchased in the checkout/payment UI
